uploadedimage/uploadedimage.go

```diff
@@ -0,0 +1,35 @@
+package uploadedfile
+
+type UplodedFile struct {
+    filename string
+    path     string
+    mime     string
+}
+
+func (this *UplodedFile) GetFilename() string {
+    return this.filename
+}
+
+func (this *UplodedFile) GetPath() string {
+    return this.path
+}
+
+func (this *UplodedFile) GetMime() string {
+    return this.mime
+}
+
+func (this *UplodedFile) FileSize() (int, error) {
```

Comment:
Opps sorry, merged before i saw this. My thought was this would be changing as i resize it. so i will be recalculating